pkgs/development/python-modules/pyside6-tools/default.nix

```diff
@@ -0,0 +1,52 @@
+{ buildPythonPackage, wrapPython, python, fetchurl, lib, stdenv, cmake, qt6,
+  shiboken6, pyside6 }:
+
+stdenv.mkDerivation rec {
+  pname = "pyside6-tools";
+
+  inherit (pyside6) version src;
+
+  patches = [
+    # Upstream has a crazy build system only geared towards producing binary
+    # wheels distributed via pypi.  For this, they copy the `uic` and `rcc`
+    # binaries to the wheel.
+    #./remove_hacky_binary_copying.patch
+  ];
+
+  postPatch = ''
+    cd sources/${pname}
```

Comment:
This should probably be replaced with sourceRoot.

Reply:
yepp, also in qtbase ... WIP